pkgs/tools/misc/3mux/default.nix

```diff
@@ -21,4 +23,4 @@ buildGoModule rec {
     # TODO: fix modules build on darwin
     broken = stdenv.isDarwin;
```

Comment:
For some reason, `3mux` builds on darwin but cross-compiling to darwin from linux doesn't work. I think this is `3mux`'s fault somehow

Reply:
@aaronjanse Thanks for letting me know.

@GrahamcOfBorg build _3mux